projects/core/src/occ/adapters/checkout/converters/occ-order-normalizer.ts

```diff
@@ -1,14 +1,17 @@
 import { Injectable } from '@angular/core';
-import { Occ } from '../../../occ-models/occ.models';
+import { Order, OrderEntry } from '../../../../model/order.model';
+import { ReplenishmentOrder } from '../../../../model/replenishment-order.model';
+import { PRODUCT_NORMALIZER } from '../../../../product/connectors/product/converters';
 import {
   Converter,
   ConverterService,
 } from '../../../../util/converter.service';
-import { Order, OrderEntry } from '../../../../model/order.model';
-import { PRODUCT_NORMALIZER } from '../../../../product/connectors/product/converters';
+import { Occ } from '../../../occ-models/occ.models';
 
 @Injectable({ providedIn: 'root' })
-export class OccOrderNormalizer implements Converter<Occ.Order, Order> {
+export class OccOrderNormalizer
+  implements
+    Converter<Occ.Order | Occ.ReplenishmentOrder, Order | ReplenishmentOrder> {
   constructor(private converter: ConverterService) {}
 
   convert(source: Occ.Order, target?: Order): Order {
```

Comment:
I we are using this normalizer shouldn't this be:

`convert(source: Occ.Order | Occ.ReplenishmentOrder, target?: Order | ReplenishmentOrder): Order | ReplenishmentOrder`?

Reply:
as agreed upon, will create a new normalizer instead